Source/Module/Login/LoginViewModel.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  LoginViewModel.swift
+//  GithubNotifications
+//
+//  Created by Chananchida F. on 9/13/21.
+//  Copyright © 2021 Nimblehq. All rights reserved.
+//
+
+import Foundation
+import SwiftUI
+import Combine
+
+protocol LoginViewModelProtocol {
+
+    func getUser(token: String, completion: @escaping CompletionHandler)
+}
+
+final class LoginViewModel: ObservableObject, LoginViewModelProtocol {
+
+    private var userService: UserServiceProtocol
+
+    var userSubscription: AnyCancellable?
+
+    init(userService: UserServiceProtocol) {
+        self.userService = userService
+    }
+    
+    func getUser(token: String, completion: @escaping CompletionHandler) {
+        UserSessionStorage.shared.authToken = token
+        userSubscription = userService.getUser()
+            .sink(
+                receiveCompletion: { completion in
+                    print(completion)
```

Comment:
I think we don't need this line but it will be useful if we want to show error when login return error so please add a `#warning`